node-zerox/src/index.ts

```diff
@@ -43,16 +44,30 @@ export const zerox = async ({
   // Download the PDF. Get file name.
   const localPath = await downloadFile({ filePath, tempDir: tempDirectory });
   if (!localPath) throw "Failed to save file to local drive";
+
+  const fileExtension = path.extname(localPath).toLowerCase();
+  let pdfPath: string;
+  const supportedExtensions = [".docx", ".pptx", ".ppt"];
+
+  // Convert file to PDF if necessary
+  if (fileExtension === ".pdf") {
+    pdfPath = localPath;
+  } else if (supportedExtensions.includes(fileExtension)) {
+    pdfPath = await convertDocxToPdf({ localPath, tempDir: tempDirectory });
+  } else {
+    throw new Error(`Unsupported file type: ${fileExtension}`);
```

Comment:
What if the file type is png, should we throw an error. Seems like we might want to add an if statement so that if it is png we can just skip the conversion steps and go straight to processing it:

```
  if (fileExtension !== ".png") {
    if (fileExtension === ".pdf") {
    } else {
    }
    await convertPdfToImages({ localPath: pdfPath, tempDir: tempDirectory });
  }
  ...
```